src/containers/find-offer/offer-container/OfferContainer.tsx

```diff
@@ -14,25 +15,29 @@ const OfferContainer: FC<OfferContainerProps> = ({ viewMode, offerCards }) => {
     console.log(id)
   }
 
-  const arrayOfCards = offerCards.map((el) => (
+  const columnNumber = viewMode === CardsViewEnums.Grid ? 12 : 1
```

Comment:
Provide adaptivity for your component. it should show only inline on tablet and only square in mobile